### PR TITLE
feat: add `localModelRootDirectory` options

### PR DESCRIPTION
**Pull request by luoye-fe**
Some ideas are for reference only 😆

### CODE REVIEW

File: src/EfficientNetCheckPointFactory.ts [hunk 1]
```diff
@@ -6,11 +6,22 @@ const defaultModelsUrl =
...
   static async create(
-    checkPoint: EfficientNetCheckPoint
+    checkPoint: EfficientNetCheckPoint,
+    options?: EfficientNetCheckPointFactoryOptions
   ): Promise<EfficientNetModel> {
-    const modelPath = `${defaultModelsUrl}${checkPoint}/${modelFileName}`;
+    const { localModelRootDirectory } = options || {};
+    let modelPath = `${defaultModelsUrl}${checkPoint}/${modelFileName}`;
+
+    if (localModelRootDirectory) {
```
Comment (ntedgi) [inline src/EfficientNetCheckPointFactory.ts:21]: this want work please follow this documentation on tensorflowJs

https://js.tensorflow.org/api/latest/#loadGraphModel

File: src/EfficientNetCheckPointFactory.ts [hunk 2]
```diff
@@ -6,11 +6,22 @@ const defaultModelsUrl =
 const modelFileName = "model.json";
 const inputLayerImageSize = [224, 240, 260, 300, 380, 456, 528, 600];
 
+interface EfficientNetCheckPointFactoryOptions {
```
Comment (ntedgi) [inline src/EfficientNetCheckPointFactory.ts:9]: move this to a separate file, please

**Review by ntedgi — COMMENTED** [review]
can you please provide a minimal unit test with the CheckPoint B0  model

File: src/EfficientNetCheckPointFactory.ts (hunk 2 above)
Comment (ntedgi) [reply src/EfficientNetCheckPointFactory.ts:12]: > can you please provide a minimal unit test with the CheckPoint B0 model
> 
> ```
> 
> test("EfficientNetCheckPointFactory - load model from custom url ", (done) => {
>   EfficientNetCheckPointFactory.create(EfficientNetCheckPoint.B0{
>     localModelRootDir: "....",
>   })
>     .then(async (model) => {
>       expect(model).toBeDefined();
>       const image = "samples/car.jpg";
>       model.inference(image).then((predictions) => {
>         expect(predictions.result[0].label).toEqual("sports car, sport car");
>         done();
>       });
>     })
>     .catch((error) => done(error));
> });
> ```
> 
> this PR close #28 #6

@luoye-fe

**Comment by luoye-fe** [comment]
emmm, bash reports an error on windows, let me fix it~

**Review by ntedgi — APPROVED** [review]

**Pull request merged**